remoting/subsystem/src/main/java/org/jboss/as/remoting/HttpConnectorAdd.java

```diff
@@ -72,4 +81,16 @@ void launchServices(OperationContext context, String connectorName, ModelNode mo
         final String saslAuthenticationFactory = saslAuthenticationFactoryModel.isDefined() ? saslAuthenticationFactoryModel.asString() : null;
         RemotingHttpUpgradeService.installServices(context, connectorName, connectorRef, RemotingServices.SUBSYSTEM_ENDPOINT, optionMap, securityRealm, saslAuthenticationFactory);
     }
+
+    void validateDuplicatedResources(OperationContext context, PathAddress address, String connectorName, Resource resource) throws OperationFailedException {
+        Resource parent = context.readResourceFromRoot(address.getParent(), true);
```

Comment:
This will have a big performance impact.
I think without recursively reading it, it should still work fine for this use case.